torchtext/prototype/models/t5/bundler.py

```diff
@@ -142,14 +142,25 @@ def build_model_from_huggingface_ckpt(
         freeze_model: bool = False,
         strict: bool = True,
     ) -> T5Model:
+        """Build T5Model model from a HuggingFace checkpoint.
+
+        Note: Only works with Huggingface models saved in the PyTorch format. Will not work \
```

Comment:
nit: do we need the "\" in the docstrings?

Reply:
Sorry - not entirely sure what you're referring to here?

Reply:
Sorry updated the comment. Looks like backslash characters don't show up when you put quotes around them which was a noob mistake 😅